shell/platform/fuchsia/flutter/tests/flatland_platform_view_unittest.cc

```diff
@@ -1093,52 +1124,157 @@ TEST_F(FlatlandPlatformViewTests, RequestFocusTest) {
   EXPECT_TRUE(focuser.request_focus_called());
 }
 
-// This test makes sure that the PlatformView correctly replies with an error
-// response when a View.focus.request call fails.
-TEST_F(FlatlandPlatformViewTests, RequestFocusFailTest) {
+// This test tries to set focus on a view without creating it first
```

Comment:
I'm a little confused because the comment says the focus work does'nt wait for view creation, but we have a view create call below. Add a comment in the code where there's a deliberately missing synchronization step?

Oh, it looks like the "View.create" platform message needs to precede the focus operation. I guess that can happen as a result of machinations on sdk/dart/fuchsia_scenic_flutter side? Like, tell us how the FuchsiaViewConnection machinery can boot up its state (like view-id) without calling a View.create message and immediately call focusByViewid().